db/migrate/20220707165348_add_user_ref_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, foreign_key: true
+  end
+end
```

Comment:
- [ ] It should be the author, not the user. Kindly change it.

You can do it like in the picture.
![image](https://user-images.githubusercontent.com/38635158/177837184-ec62ca8a-3a06-4c56-b300-fa2820ebe958.png)